tests/test_tensors.py

```diff
@@ -0,0 +1,176 @@
+import numpy as np
+
+import pytest
+
+from devito import VectorFunction, TensorFunction, VectorTimeFunction, TensorTimeFunction
+from devito import Grid, Function, TimeFunction, Dimension, Eq
+from devito.types import NODE
+from devito.finite_differences.differentiable import Add
+
+
+def dimify(dimensions):
```

Comment:
ugly?

Reply:
That's how it is in every other test that needs dimensions